sonar-scm-svn-plugin/src/main/java/org/sonar/plugins/scm/svn/SvnScmProvider.java

```diff
@@ -99,21 +99,27 @@ public BlameCommand blameCommand() {
     return null;
   }
 
-  private static Set<Path> computeChangedPaths(Path rootBaseDir, SVNClientManager clientManager) throws SVNException {
+  private static Set<Path> computeChangedPaths(Path projectBasedir, SVNClientManager clientManager) throws SVNException {
     SVNWCClient wcClient = clientManager.getWCClient();
-    SVNInfo svnInfo = wcClient.doInfo(rootBaseDir.toFile(), null);
-    String base = "/" + Paths.get(svnInfo.getRepositoryRootURL().getPath()).relativize(Paths.get(svnInfo.getURL().getPath()));
+    SVNInfo svnInfo = wcClient.doInfo(projectBasedir.toFile(), null);
+
+    // SVN path of the repo root, for example: /C:/Users/JANOSG~1/AppData/Local/Temp/x/y
+    String svnRootPath = svnInfo.getRepositoryRootURL().getPath();
```

Comment:
Why not converting the URL to Path directly, to avoid manipulating strings? Like `Paths.get(svnInfo.getRepositoryRootURL().toURI())` ?

Reply:
An `SVNURL` (returned by `svnInfo.getRepositoryRootURL()`) doesn't have a `toURI()` method, only the `getPath()`, which on Windows returns paths like the example in the comment above. `Paths.get(...)` chokes on such paths with:

>     java.nio.file.InvalidPathException: Illegal char <:> at index 2: /C:/Users/JANOSG~1/AppData/Local/Temp/junit5258881857313956908/junit176566423469811967

I'm open to better ways than string manipulation, but I don't see any...